### PR TITLE
Make zip command path configurable

### LINKED ISSUE #413: zip command used in shapefile export should be configurable
It may be located in a non-standard place

It has a `FIXME` warning in the code :smile: https://github.com/CartoDB/CartoDB-SQL-API/blob/master/app/models/formats/ogr/shp.js#L36

### PR DESCRIPTION
**Pull request by dgaubert**
Closes #413

### CODE REVIEW

File: app/models/formats/ogr/shp.js [hunk 1]
```diff
@@ -63,7 +63,7 @@ ShpFormat.prototype.toSHP = function (options, callback) {
         //console.log("Zip complete, zip return code was " + code);
         if (code) {
           next(new Error("Zip command return code " + code));
-          //res.statusCode = 500; 
+          //res.statusCode = 500;
```
Comment (ethervoid) [inline app/models/formats/ogr/shp.js]: I'd make some clean in the class like remove commented code like this

**Comment by ethervoid** [comment]
Applying the patch, the command works so good for me :)

File: app/models/formats/ogr/shp.js [hunk 2]
```diff
@@ -63,7 +63,7 @@ ShpFormat.prototype.toSHP = function (options, callback) {
...
         }
         next(null);
```
Comment (dgaubert) [inline app/models/formats/ogr/shp.js]: OMG! This is an anti-pattern, `next` callback is actually called twice. Fixing it...

File: app/models/formats/ogr/shp.js (hunk 1 above)
Comment (dgaubert) [inline app/models/formats/ogr/shp.js]: 👍

**Comment by dgaubert** [comment]
I've seen a potential bug, please @ethervoid, could review it again?

**Pull request merged**